input/pages/changes.md

```diff
@@ -3,8 +3,13 @@
 {: #changes}
 
 This page details changes made in each version of the Using CQL with FHIR Implementation Guide.
+### STU1 Reconciliation Changes (version 1.0.0-ballot)
 
-### Initial STU Ballot (version 1.0.0-ballot)
+Change Summary
+This ballot made the following major changes:
+
+
+### Initial STU 1 Ballot Changes (version 1.0.0-ballot)
```

Comment:
I think we need to move what's currently in the package-list.json changes section and put it here, right?

Reply:
Moved changes listed in package-list.yml to changes.md.